odl/test/largescale/tomo/analytic_slow_test.py

```diff
@@ -174,7 +174,8 @@ def test_fbp_reconstruction(projector):
     # Compute the FBP result
     fbp_result = fbp_operator(projections)
 
-    maxerr = vol.norm() / 5.0
+    # Allow 30 % error
+    maxerr = vol.norm() * 0.3
```

Comment:
why increase?

Reply:
I'll have to check that again, it sometimes failed with the lower error margin (also on master).